CHANGELOG.md

```diff
@@ -21,19 +23,27 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
     - This fixes odd issues which can occur when mounting over an existing
       mount point (we mount `.` as `/opt/openoni`, and previously were mounting
       `./docker/data` as `/opt/openoni/data`, effectively shadowing the actual data directory)
-  - All generated ingest artifacts now live in a named volume
+  - All generated ingest artifacts now live in a named volume, `onidata`, which
```

Comment:
I think this should note that it only relates to a Docker (Compose) environment.

Reply:
That's literally the parent of the bullet point in question :)

Reply:
Ah okay. I didn't see that as I was just looking at the changes without the full file for context. :+1: